File: test/cpp-examples/Hello.cpp

class Hello {
  public:
    int pow2 (int n);
    Hello();
    ~Hello();
};

Hello::Hello () {};
Hello::~Hello () {};

int
Hello::pow2 (int n)
{
  return (1 << n);
}


int main ()
{
  Hello* hello = new Hello();
  hello->pow2(100);
  hello->pow2(5);
  
  return 0;
}

File: jvm/test/Hello.java

public class Hello{
	String name;
	int year;
	
	//calculate 2^n
	int pow2(int n) {
	 return (1<<n);
	}
	
	void sayHello(String name) {
	    System.out.println("Hello " + name);
	}
	
	public static void main(String[] zArgs){
		int k;
		Hello hello = new Hello();
		hello.year = 2013;
		k = hello.pow2(7);
		hello.sayHello("Anass");
	}
}
